### DISCUSSION TITLE
How to get FullXPath Value?

### DISCUSSION THREAD

**Discussion by kbpillai** [GoogleChrome/lighthouse#15675] [Q&A] [answered]
In Lighthouse JSON, the selector value seems not to be accurate all the time. the PATH comes as "2,HTML,0,HEAD,3,SCRIPT" etc., Is there a way to retrieve the actual XPATH or FullXPath in JSON?

**Comment by adamraine** [comment]
No, we don't surface the node path in the xpath format.

**Comment by kbpillai** [comment]
Is there a way to retrieve the XPath for that element?

**Comment by kbpillai** [comment, marked as answer]
Thank you!

 

Paul Irish" ***@***.***> wrote:

 

This method translates the node path to an XPATH: https://source.chromium.org/search?q=f:node.cc%20symbol:PrintNodePathTo&sq=&ss=chromium%2Fchromium%2Fsrc
Obviously you can't use it directly but it shouldn't be too bad to port.
